internal/pkg/deploy/cloudformation/stack/backend_svc.go

```diff
@@ -19,6 +19,11 @@ const (
 	BackendServiceContainerPortParamKey = "ContainerPort"
 )
 
+const (
+	// BlankBackendSvcContainerPort indicates no port is exposed for service container.
+	BlankBackendSvcContainerPort = "-1"
```

Comment:
nit: So that the the const can be used for other service types
```suggestion
	// NoExposedContainerPort indicates no port should be exposed for the service container.
	NoExposedContainerPort = "-1"
```